Project_Proposal.md

```diff
@@ -0,0 +1,30 @@
+### Name of the game: 
+# Brain on blue
+
+### Short description of gameplay: 
+Side scrolling  adventure game of a brain named BOB.
+Bob is searching for a head to inhabit but must travel
+overcome various obstacles to eventually find the head.
+
+### Win condition:
+BOB is able to navigate past all obsolesces and find a head
+
+### Lose condition:
+Obstacles cause enough damage that BOB dies
+
+### How the player interacts with the game:
+the interface will be via the keyboard using the "up", "Down",
+"left", and right arrow keys.
+
+### List of game features:
+* side scrolling adventure
+* interesting obstacles to navigate with left right and jump
+* brain stem is an attack, "down key"
+* at least two levels , and second level is "moving"faster
+
+### Realistic goals/expectations for 4 weeks of work on the project:
+without a "cool background, it should be do-able, the level may be 
+the only possible constraint.
```

Comment:
Making levels doesn't have to be too hard, I'll make sure we talk about it soon.

Reply:
I feel i might need to go back over level creation when i get to thet part of development